packages/gen-typescript-declarations/src/gen-ts.ts

```diff
@@ -231,7 +231,7 @@ function addAutoImports(tsDoc: ts.Document, autoImport: Map<string, string>) {
   const alreadyImported = getImportedIdentifiers(tsDoc);
 
   for (const node of tsDoc.traverse()) {
-    if (node.kind !== 'name') {
+    if (node.kind !== 'name' && node.kind !== 'param') {
```

Comment:
How does this work? Why does skipping params mean that they get autoimported?

Reply:
It's negated, so it's included, not skipped, I'll rewrite it to be less confusing.

Reply:
Done.